src/query/block/column.go

```diff
@@ -115,23 +117,34 @@ func (c *colBlockIter) StepCount() int {
 
 // Next returns true if iterator has more values remaining
 func (c *colBlockIter) Next() bool {
```

Comment:
maybe write comments regarding the flow ? Eg: check for Next() if that's false, then also check for Err() ?

Reply:
SGTM. Considering that these are interface methods though, might get rid of the comments here and make the iterator ones more meaningful